src/applications/check-in/pages/CheckIn.jsx

```diff
@@ -1,24 +1,48 @@
-import React from 'react';
-
-import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment-timezone';
 
 import { goToNextPageWithToken, getTokenFromRouter } from '../utils/navigation';
 
 import { checkInUser } from '../api';
 
+import BackToHome from '../components/BackToHome';
+import Footer from '../components/Footer';
+
 const CheckIn = props => {
-  const { router } = props;
+  const { router, appointment } = props;
+  const [isLoading, setIsLoading] = useState(false);
+
+  if (!appointment) {
+    goToNextPageWithToken(router, 'failed');
+    return <></>;
+  }
+
   const token = getTokenFromRouter(router);
   const onClick = async () => {
-    const json = await checkInUser({ some: 'data', token });
+    setIsLoading(true);
+    const json = await checkInUser({
+      token,
+    });
     const { data } = json;
-    if (data.status === 'checked-in') {
+    if (data.checkInStatus === 'completed') {
       goToNextPageWithToken(router, 'confirmed');
     } else {
       goToNextPageWithToken(router, 'failed');
     }
   };
-  const contactNumber = '555-867-5309';
+
+  const appointmentDate = moment(new Date(appointment.appointmentTime)).format(
```

Comment:
i assume this is all temporary scaffolding that will come out when when we have actual appointment information to propagate up from CHIP through vets-api?

Reply:
100%! For UAT all the mock data was being generated client-side, now the vets-api is supplying the mock data. Once CHIP gets set up we should be seeing real data.